client/src/styles/listing.css

```diff
@@ -1,8 +1,8 @@
-body {
+.body-font {
   font-family: 'Montserrat', sans-serif;
 }
 
-img {
+.similar {
```

Comment:
@VALE46GP I would make this a more descriptive class name. Ie. what does this css class do and on what type of object.